<robot name="tf_simulation1">

    <link name="world" />
    <link name="base_link" />
    <link name="body1_link" />
    <link name="body2_link" />

    <joint name="world_to_base_joint" type="fixed">
        <origin xyz="0 0 0" rpy="0 0 0"/>
        <parent link="world"/>
        <child link="base_link"/>
    </joint>
    <joint name="base_to_body1_joint" type="fixed">
        <origin xyz="0 0 0" rpy="0 0 0"/>
        <parent link="base_link"/>
        <child link="body1_link"/>
    </joint>
    <joint name="body1_to_body2_joint" type="fixed">
        <origin xyz="0 0 0" rpy="0 0 0"/>
        <parent link="body1_link"/>
        <child link="body2_link"/>
    </joint>

</robot>